File: VB/SchedulerMappedLabels/Form1.vb
Imports System
Imports System.Windows.Forms

Namespace SchedulerMappedLabels

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs)
            ' TODO: This line of code loads data into the 'carsDBDataSet.CarScheduling' table. You can move, or remove it, as needed.
            carSchedulingTableAdapter.Fill(carsDBDataSet.CarScheduling)
            ' TODO: This line of code loads data into the 'carsDBDataSet.Labels' table. You can move, or remove it, as needed.
            labelsTableAdapter.Fill(carsDBDataSet.Labels)
            schedulerControl1.PopulateLabelsStorage()
            If schedulerControl1.Storage.Appointments.Count > 0 Then
                Dim start As Date = schedulerControl1.Storage.Appointments(0).Start
                schedulerControl1.Start = start.Date
                schedulerControl1.DayView.TopRowTime = start.TimeOfDay
            End If
        End Sub
    End Class
End Namespace

File: CS/SchedulerMappedLabels/Form1.cs
using System;
using System.Windows.Forms;

namespace SchedulerMappedLabels {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            // TODO: This line of code loads data into the 'carsDBDataSet.CarScheduling' table. You can move, or remove it, as needed.
            this.carSchedulingTableAdapter.Fill(this.carsDBDataSet.CarScheduling);
            // TODO: This line of code loads data into the 'carsDBDataSet.Labels' table. You can move, or remove it, as needed.
            this.labelsTableAdapter.Fill(this.carsDBDataSet.Labels);


            if (schedulerControl1.DataStorage.Appointments.Count > 0) {
                DateTime start = schedulerControl1.DataStorage.Appointments[0].Start;
                schedulerControl1.Start = start.Date;
                schedulerControl1.DayView.TopRowTime = start.TimeOfDay;
            }
        }
    }
}